lib/rev-api/models/order_request.rb

```diff
@@ -169,6 +194,60 @@ class Input < ApiSerializable
 
     # External URL, if sources wasn't POSTed as input (YouTube, Vimeo, Dropbox, etc)
     attr_reader :external_link
+
+    # Optional, list of glossary entries.
+    attr_reader :glossary
+
+    # Optional, list of speaker names.
+    attr_reader :speakers
+
+    # Optional, list of accents.
+    attr_reader :accents
+
+    SUPPORTED_ACCENTS = {
+      :american_neutral => 'AmericanNeutral',
+      :american_southern => 'AmericanSouthern',
+      :asian => 'Asian',
+      :australian => 'Australian',
+      :british => 'British',
+      :indian => 'Indian',
+      :other => 'Other',
+      :unknown => 'Unknown'
+    }
+
+    def validate_glossary
+      if glossary
+        if glossary.length > GLOSSARY_ENTRIES_LIMIT
+          raise(ArgumentError, "Glossary must not exceed #{GLOSSARY_ENTRIES_LIMIT} entries")
+        end
+        glossary.each { |term|
+          if term.length > GLOSSARY_ENTRY_LENGTH_LIMIT
+            raise(ArgumentError, "Glossary entries cannot exceed #{GLOSSARY_ENTRY_LENGTH_LIMIT} characters")
+          end
+        }
+      end
+    end
+
+    def validate_speakers
+      if speakers
+        if speakers.length > SPEAKER_ENTRIES_LIMIT
+          raise(ArgumentError, "Speaker list must not exceed #{SPEAKER_ENTRIES_LIMIT} entries")
+        end
+        speakers.each { |speaker|
+          if speaker.length > SPEAKER_ENTRY_LENGTH_LIMIT
+            raise(ArgumentError, "Speaker name cannot exceed #{SPEAKER_ENTRY_LENGTH_LIMIT} characters")
+          end
+        }
+      end
+    end
+
+    def validate_accents
```

Comment:
should validate that the accents array does not contain more entries than the amount of accents we do support